packages/studio-base/src/panels/ThreeDeeRender/ModelCache.ts

```diff
@@ -33,6 +35,8 @@ type LoadModelOptions = {
   referenceUrl?: string;
 };
 
+type CacheKey = string & { __brand: "CacheKey" };
```

Comment:
```suggestion
type CacheKey = Opaque<string, "CacheKey">;
```

https://github.com/foxglove/studio/blob/aeae9afbc04a15cc6a854d4ea1e138f00296e835/packages/studio-base/src/context/PerformanceContext.ts#L15